src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,40 @@
+import { Status } from '../types/Status';
+import { Todo } from '../types/Todo';
+import { TodosFilter } from './TodosFilter';
+
+type Props = {
+  unCompletedTodos: number,
+  filter: Status,
+  setFilter: React.Dispatch<React.SetStateAction<Status>>
+  todos: Todo[],
+  removeCompletedTodos: () => void
+};
+
+export const TodoFooter:React.FC<Props> = ({
+  unCompletedTodos,
+  filter,
+  setFilter,
+  todos,
+  removeCompletedTodos,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${unCompletedTodos} items left`}
```

Comment:
It's better to keep the UI text related to pluralization grammatically correct. For instance, '1 item left' instead of '1 items left'. You can include a condition to check if 'unCompletedTodos' is 1 and accordingly display 'item' or 'items'.